### PR TITLE
envv: New formula

### PR DESCRIPTION
**Pull request by bukzor**
test and audit pass

This project hasn't changed in a few years, but it's been around since 1994, and the maintainers are responsive whenever I need something.

### CODE REVIEW

File: Library/Formula/envv.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
```
Comment (DomT4) [inline Library/Formula/envv.rb]: You can cut these two lines ;)

Comment (bukzor) [inline Library/Formula/envv.rb]: Done. Might should check this in `brew audit`.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `envv 1.7 (new formula)`.

File: Library/Formula/envv.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/envv.rb]: This line will be flagged in ` brew audit ` in the future :+1:

**Pull request closed without merging**